exporter/elasticexporter/go.mod

```diff
@@ -11,6 +11,8 @@ require (
 	go.uber.org/zap v1.19.1
 )
 
+require go.uber.org/multierr v1.6.0
```

Comment:
any reason for the different version here?

Reply:
That's what `make gotidy` did, but I manually fix